### ISSUE TITLE
isNaN for compexes. 

### ISSUE THREAD

**Issue by balagge** [josdejong/mathjs#930]
I do not know if this is a bug or some intended behavior, but I guess it is a bug.
The utility function `isNaN()` works for Complexes:
```
'Complex': function (x) {
      return Number.isNaN(x.re) && Number.isNaN(x.im);
    },
```

Here, the `&&` should be `||`.

Better still, complex.js offers an `isNaN()` method (which returns `isNaN(this['re']) || isNaN(this['im'])`) btw.

So it could also be:

```
'Complex': function (x) {return x.isNaN();}
```

**Comment by HarrySarson** [comment]
That looks like a bug to me

**Comment by josdejong** [comment]
Thanks for reporting, this is indeed a bug. It's fixed now in the `develop` branch.

**Comment by josdejong** [comment]
Should be fixed now in `v3.16.3``
